### PR TITLE
feat: add "npm org" command

### PR DESCRIPTION
**Pull request by chrisdickinson**
This adds support for adding, removing, and listing org memberships from the command line!

```
$ npm org add npmcorp chrisdickinson admin
$ npm org set npmcorp chrisdickinson developer
$ npm org rm npmcorp chrisdickinson
$ npm org ls npmcorp
```

### CODE REVIEW

**Comment by zkat** [comment]
@chrisdickinson Could you add some tests? https://github.com/npm/npm/blob/latest/test/tap/team.js is the corresponding test for team.

**Review by zkat — APPROVED** [review]
LGTM 👍

**Review by iarna — CHANGES_REQUESTED** [review]
I'm of the opinion that cli commands should never emit JSON unless explicitly asked to with `--json` and that we should come up with some human readable output format for these commands.

There also need to be docs for these new commands.

File: lib/org.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+function org (args, cb) {
+  // Entities are in the format <scope>:<org>
+  var cmd = args.shift()
+  var orgname = args.shift()
+  var username = args.shift()
+  var role = args.shift()
+  return mapToRegistry('/', npm.config, function (err, uri, auth) {
+    if (err) { return cb(err) }
+    try {
```
Comment (watilde) [inline lib/org.js]: Probably, we can get rid of this `try-catch` like the other implementations.

File: node_modules/npm-registry-client/index.js [hunk 2]
```diff
@@ -60,6 +60,7 @@ function RegClient (config) {
   client.get = require('./lib/get')
   client.initialize = require('./lib/initialize')
   client.logout = require('./lib/logout')
+  client.org = require('./lib/org')
```
Comment (watilde) [inline node_modules/npm-registry-client/index.js]: Somehow we're missing this module file. Let's run:
> $ scripts/dep-update npm-registry-client 8.2.0

File: lib/config/cmd-list.js [hunk 3]
```diff
@@ -32,6 +32,7 @@ var affordances = {
   'login': 'adduser',
   'add-user': 'adduser',
   'author': 'owner',
+  'ogr': 'org',
```
Comment (zkat) [inline lib/config/cmd-list.js]: 👹

Comment (watilde) [reply lib/config/cmd-list.js:35]: might be fine as an affordance?

**Comment by zkat** [comment]
Hi! We're moving repos to https://github.com/npm/cli/pulls! See [our blog post about the migration to npm.community](https://blog.npmjs.org/post/175587538995/announcing-npmcommunity) for details. As such, we're closing all active PRs on this repo.

Could you please re-open this PR against `npm/cli` instead? We're still interested in this patch so we hope you will! We'll continue discussion over there once it's moved. 💚

**Comment by zkat** [comment]
Additionally, chris, a new version of this might want to follow the implementation of `npm hook`, and use `npm-registry-fetch` plus a `libnpmorg` or something.

**Pull request closed without merging**